### ISSUE TITLE
Wrong measurement of option line length

### ISSUE THREAD

**Issue by dtu-compute** [dtu-compute/dtu-quiz#218]
On the below screenshot is `12` on the selected line is wrapped from the end to the beginning of the line.

An interesting note is that, if the longest line is made shorter than the second longest line, then the last part of the now longest line is wrapped. So somehow the point where it wraps the line is relative and not a fixed length.

The quiz code is also below.

![lhiljlfijiadmibk](https://cloud.githubusercontent.com/assets/13518486/22519342/45b24dca-e8b0-11e6-95cf-c10a5973e943.png)

```
begin:multiple_choice
begin:text

Boxplots for 3 different groups of data with $n=20$ data points in each group are shown below. (It can be assumed that there are no ''ties'', that is no identical observations.)

![Boxplot](http://www2.compute.dtu.dk/courses/introstat/images/w1_boxplot_800.png)

Which of the following statements is _false_?
end:text

	wrong "The median for the data in b is approximately 8"
	wrong "The largest observation among all 3 groups of data is approximately $12$"
	wrong "The data in boxplot c has the smallest variance"
	begin:right
		begin:option
			The 90th percentile for the data in boxplot a is approximately $12$
		end:option
		begin:feedback
		
			If there is no outliers, then the outer limits in the boxplot marks the minimum and maximum values. When there is one or more outliers, then the outer limits mark the most extreme vaules in the data still WITHIN $1.5\times IQR$ of the box itself. So in boxplot a, observation nr.\ 19 is around 10. But the 90\% percentile would among $n=20$ observations be the average of the $18$th and $19$th observation - hence less than 10, so it cannot be 12.
			
						
		end:feedback
	end:right
	wrong "The third quartile for boxplot c is approximately 7.4"
	  
end:multiple_choice
```

**Comment by dtu-compute** [comment]
With the latest commit it looks a lot better. Now it breaks to a new line. As there is more space on screen it would be preferred if it wasn't broken to next line.

In the second picture is a newline inserted somehow.

![1](https://cloud.githubusercontent.com/assets/13518486/22589532/077597cc-ea0b-11e6-8438-dc41f5175496.png)

![2](https://cloud.githubusercontent.com/assets/13518486/22589539/0d443dc0-ea0b-11e6-8427-78f220d0d760.png)

**Comment by iainbryson** [comment]
Fixed (by fixing #230, I think)

![image](https://cloud.githubusercontent.com/assets/1531103/23049821/8160fd2a-f473-11e6-9293-f3d589e9113d.png)
